src/liquid/components/ld-bg-cells/readme.md

```diff
@@ -90,55 +91,55 @@ A background pattern with the Merck cells as additional visual element.
 <!-- <div class="ld-bg-cells ld-bg-cells--bioreliance box2-16/9">
   <div class="ld-bg-cells__pattern"></div>
   <div class="ld-bg-cells__content">
-    <p class="ld-paragraph">type: bioreliance</p>
+    <p class="ld-typo--body-m">type: bioreliance</p>
```

Comment:
I know how bem works, but adding the modifier is sufficient in this case and adding an extra class just to keep BEM up would not make sense to me. It's just the way it was with the ld-paragraph component, as well. Though I think, I added the font style to the ld-typo, as well, in the meanwhile, as "body-m" is now the default variant.

Reply:
If `body-m` is the default, then I would change the style definition from
```css
.ld-typo,
.ld-typo--body-m {
  font: var(--ld-typo-body-m);
}
```
to
```css
.ld-typo {
  font: var(--ld-typo-body-m);
}
```
and get rid of `.ld-typo--body-m` because it is not a modifier.

Reply:
As mentioned yesterday, let's keep the modifier, as every other typo variant does have one, too. But we can update the code in the README to use the .ld-typo class, instead.